lib/config/hadeployconfigverify.go

```diff
@@ -14,6 +19,12 @@ func (c *HaDeployConfig) Verify() error {
 
 	// Validate ExistingInfra (on prem)
 	if c.IsExistingInfra() {
+		if c.ObjectStorage != nil && c.ObjectStorage.Config.Location == GCS_STORAGE {
+			err := c.readGCSConfigFile(c.ObjectStorage.Config)
+			if err != nil {
+				return err
```

Comment:
Can we move this into the following function in line no - 191: 
https://github.com/chef/automate/blob/e2fbc14136df0e5472c24e5702e6a48f82079bc6/lib/config/hadeployconfigverify.go#L191C11-L191C11

Function name and signature: verifyObjectStorage(objectStorage *ConfigObjectStorage) error